Client/Frontend/Browser/Tab Tray/TabTrayController.swift

```diff
@@ -32,13 +32,40 @@ class TabTrayController: LoadingViewController {
     case local
     case sync
   }
+  
+  // MARK: SyncActionState
+  
+  enum SyncActionState {
+    case noSyncChain
+    case openTabsDisabled
+    case noSyncedSessions
+    case activeSessions
+  }
 
   let tabManager: TabManager
   private let braveCore: BraveCoreMain
   private var openTabsSessionServiceListener: OpenTabsSessionStateListener?
   
   weak var delegate: TabTrayDelegate?
   weak var toolbarUrlActionsDelegate: ToolbarUrlActionsDelegate?
+  
+  private var emptyPanelState: SyncActionState {
```

Comment:
Is this really an empty state getter now? the `. activeSessions` enum value sounds like a non-empty state to me?

Reply:
Fix at https://github.com/brave/brave-ios/pull/6028/commits/4802ebe51b559a77b910c92f47ba92c7519a7a48